src/runtime/recipe/recipe-interface.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from '../../platform/assert-web.js';
+
+
+export interface RecipeInterface {
+  // removeHandle(): void;
```

Comment:
Looks like this is still in work. I wondered about naming the interface more clearly to reflect what it is/does. HandleRepository (if all it does is remove/interact with handles)... etc. To consider as you progress further.